bin/ch-tar2dir

```diff
@@ -51,7 +45,7 @@ for ext in '' .tar.gz .tar.xz .tgz; do
     fi
     tarball=$c
     if [ -n "$ext" ]; then
-        echo "found: ${tarball}" 1>&2
+        echo "found: $tarball" 1>&2
```

Comment:
Recall style is to enclose variable expansion in curly brackets unless it's the only thing in the string.